app/kubernetes/views/applications/create/createApplicationController.js

```diff
@@ -186,9 +187,16 @@ class KubernetesCreateApplicationController {
   }
   /* #endregion */
 
-  onChangeStackName(stackName) {
+  onChangeStackName(name) {
     return this.$async(async () => {
-      this.formValues.StackName = stackName;
+      if (KUBE_STACK_NAME_VALIDATION_REGEX.test(name) || name === '') {
+        this.state.stackNameError = '';
+      } else {
+        this.state.stackNameError =
+          "Stack must consist of alphanumeric characters, '-', '_' or '.', must start and end with an alphanumeric character and must be 63 characters or less (e.g. 'my-name', or 'abc-123').";
```

Comment:
Add missing `stackNameError: '',` in state definition L130
```js
       isExistingCPUReservationUnchanged: false,
       isExistingMemoryReservationUnchanged: false,
       stackNameError: '', // here
     };
```

Reply:
Updated